src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.WinRT.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
Having a common binary is interesting when you are trying to reduce what you ship / need to test.  Since this is a library that clearly has some platform specifics were already going to test it on all platforms.  Since this library is part of netstandard, therefore part of our framework packages, we already ship a copy per-platform.